src/include/sof/audio/component.h

```diff
@@ -249,6 +249,14 @@ struct comp_dev {
 	uint32_t output_rate;      /**< 0 means all output rates are fine */
 	struct pipeline *pipeline; /**< pipeline we belong to */
 
+	uint32_t min_sink_bytes;   /**< min sink buffer size measured in bytes
+				     *  required to run component's processing
```

Comment:
Min **free** sink buffer size ?

Reply:
@lgirdwood 
Yes, that's what I meant. I've updated comments.